docs/modules/mssql.md

```diff
@@ -177,4 +303,52 @@ Drops the applicable user from the SQL Server.
 
 ##### Deprovision
 
-Deletes both the database from the SQL Server instance, but does not delete the DBMS.
\ No newline at end of file
+Deletes both the database from the SQL Server instance, but does not delete the DBMS.
```

Comment:
Should this be "both **of** the database**s**"?

Reply:
Should be deletes *ONLY* the database